src/Microsoft.VisualStudio.Composition/Reflection/TypeRef.cs

```diff
@@ -65,7 +65,7 @@ public class TypeRef : IEquatable<TypeRef>, IEquatable<Type>
             Requires.NotNullOrEmpty(fullName, nameof(fullName));
 
             this.resolver = resolver;
-            this.AssemblyName = GetNormalizedAssemblyName(assemblyName);
+            this.AssemblyName = Resolver.GetNormalizedAssemblyName(assemblyName);
```

Comment:
These won't use cached versions

Reply:
Sounds good, thank you! I scanned the caller tree a bit and seemed as much, but wanted to get confirmation. Resolving!